docs/reference/release-notes/8.0.0-alpha1.asciidoc

```diff
@@ -1,44 +1,520 @@
+* Apply cluster states in system context {es-pull}53785[#53785] (issue: {es-issue}53751[#53751])
+
+Distributed::
+* Introduce ?wait_for_active_shards=index-setting {es-pull}67158[#67158] (issue: {es-issue}66419[#66419])
+* Respect `CloseIndexRequest#waitForActiveShards` in HLRC {es-pull}67374[#67374] (issues: {es-issue}67158[#67158], {es-issue}67246[#67246])
+
+Features/Java High Level REST Client::
+* Fix HLRC compatibility with Java 8 {es-pull}74290[#74290] (issues: {es-issue}73910[#73910], {es-issue}74272[#74272], {es-issue}74289[#74289])
+
+Geo::
+* Preprocess polygon rings before processing it for decomposition {es-pull}59501[#59501] (issues: {es-issue}54441[#54441], {es-issue}59386[#59386])
+
+Highlighting::
+* Bug fix for `AnnotatedTextHighlighter` - port of 39525 {es-pull}39747[#39747]
+
+Infra/Core::
+* Add searchable snapshot cache folder to `NodeEnvironment` {es-pull}66297[#66297] (issue: {es-issue}65725[#65725])
+* CLI tools: write errors to stderr instead of stdout {es-pull}45586[#45586] (issue: {es-issue}43260[#43260])
+* Precompute `ParsedMediaType` for XContentType {es-pull}67409[#67409]
+
+Infra/Logging::
+* Fix NPE when logging null values in JSON {es-pull}53715[#53715] (issue: {es-issue}46702[#46702])
+* Fix stats in slow logs to be a escaped JSON {es-pull}44642[#44642]
+* Populate data stream fields when `xOpaqueId` not provided {es-pull}62156[#62156]
+
+Infra/REST API::
+* Do not allow spaces within `MediaType's` parameters {es-pull}64650[#64650] (issue: {es-issue}51816[#51816])
+* Fix: added missing skip {es-pull}41492[#41492]
+* Handle incorrect header values {es-pull}64708[#64708] (issues: {es-issue}51816[#51816], {es-issue}64689[#64689])
+* Ignore media ranges when parsing {es-pull}64721[#64721] (issues: {es-issue}51816[#51816], {es-issue}64689[#64689])
+* `RestController` should not consume request content {es-pull}44902[#44902] (issue: {es-issue}37504[#37504])
+
+Infra/Scripting::
+* Change compound assignment structure to support String concatenation {es-pull}61825[#61825]
+* Fixes casting in constant folding {es-pull}61508[#61508]
+* Several minor Painless fixes {es-pull}61594[#61594]
+
+MULTIPLE AREA LABELS::
+* Avoid `StackOverflowError` due to regex alternate paths {es-pull}61259[#61259] (issue: {es-issue}60889[#60889])
+* Fix cat recovery display of bytes fields {es-pull}40379[#40379] (issue: {es-issue}40335[#40335])
+* Fixes to task result index mapping {es-pull}50359[#50359] (issue: {es-issue}50248[#50248])
+
+Machine Learning::
+* Checking if p-tasks metadata is null before updating state (#41091) {es-pull}41124[#41124] (issues: {es-issue}41090[#41090], {es-issue}41091[#41091])
+* Fix typo Korean HANGUL_JAMO {es-pull}74462[#74462]
+* Handle null value of `FieldCapabilitiesResponse` {es-pull}64327[#64327]
+* Relax the test assertion so that the test always passes {es-pull}44133[#44133] (issue: {es-issue}44132[#44132])
+* Remove erroneous licence {es-pull}58732[#58732] (issue: {es-issue}58682[#58682])
```

Comment:
```suggestion
```

Internal mistake that was never released - should have been labelled `>non-issue`.